packages/mui-material/src/Autocomplete/Autocomplete.test.js

```diff
@@ -202,12 +203,44 @@ describe('<Autocomplete />', () => {
           renderInput={(params) => <TextField {...params} autoFocus />}
         />,
       );
-
       checkHighlightIs(getByRole('listbox'), 'one');
       setProps({ value: 'two' });
       checkHighlightIs(getByRole('listbox'), 'two');
     });
 
+    it('should have focused on first element after navigating all elements once by keyboard', () => {
+      const { getByRole } = render(
```

Comment:
Please add
```js
if (/jsdom/.test(window.navigator.userAgent)) {
  this.skip();
}
```

to the top of the test, as it always passes in JSDOM. It only makes sense to test this on real browsers.